### PR TITLE
f-checkout - Convert component tests to visual regression tests

### PR DESCRIPTION
**Pull request by siggerzz**
This PR converts a number of component tests to use Percy visual regression, rather than WebDriverIO's `isDisplayed` function.

### CODE REVIEW

File: packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js [hunk 1]
```diff
@@ -109,9 +109,9 @@ module.exports = class Checkout extends Page {
         const isLoggedIn = `&knob-Is%20User%20Logged%20In=${checkout.isAuthenticated}`;
         const hasCheckoutErrors = `&knob-Checkout%20Errors=${checkout.checkoutErrors}`;
         const hasPlaceOrderErrors = `&knob-Place%20Order%20Errors=${checkout.placeOrderErrors}`;
-        const hasPreorderWarning = `&knob-Is%20ASAP%20available=${checkout.isPreOrderWarningDisplayed}`;
```
Comment (Preet-Sandhu) [inline packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js:112]: Is this an intentional delete?

File: packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js [hunk 2]
```diff
@@ -109,9 +109,9 @@ module.exports = class Checkout extends Page {
...
+        const hasAsapAvailable = `&knob-Is%20ASAP%20available=${checkout.isAsapAvailable}`;
 
-        const url = `checkout-component${serviceType}${isLoggedIn}${hasCheckoutErrors}${hasPlaceOrderErrors}${hasPreorderWarning}`;
```
Comment (Preet-Sandhu) [inline packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js:114]: same as above

File: packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js [hunk 3]
```diff
@@ -123,18 +123,6 @@ module.exports = class Checkout extends Page {
...
-        return this.fields[fieldName].error.isDisplayed();
-    }
-
-    isFieldDisplayed (fieldName) {
-        return this.fields[fieldName].input.isDisplayed();
-    }
-
     isPostcodeTypeErrorDisplayed () {
         return this.fields.addressPostcode.typeError.isDisplayed();
     }
```
Comment (Preet-Sandhu) [inline packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js:128]: Did you want to move all the 'is displayed' tests? Is this one left here intentionally?

**Review by LTurns — APPROVED** [review]

**Review by Preet-Sandhu — CHANGES_REQUESTED** [review]

File: packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js (hunk 1 above)
Comment (siggerzz) [reply packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js:112]: Yep. I renamed it to a more sensible variable name of `hasAsapAvailable` :)

File: packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js (hunk 2 above)
Comment (siggerzz) [reply packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js:114]: ditto

File: packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js (hunk 3 above)
Comment (siggerzz) [reply packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js:128]: This is intentional. There's a single component test left to check that error _isn't_ displayed when entering valid details. :)

**Review by Preet-Sandhu — APPROVED** [review]

**Pull request merged**